ouroboros-consensus-cardano/changelog.d/20230809_120937_nick.frisby_preparing_8_3_0_pre.md

```diff
@@ -0,0 +1,8 @@
+### Breaking
```

Comment:
In terms of code changes, this diff would be patch-level, except for the type change on `GetConstitutionHash`. Since that query is unreleased as of yet (it's Conway only), maybe `Breaking` here is excessive?

The type of `getPParams` also changes, but it's not really part of the downstream interface---more so internal.

On the other hand, we're also increasing the major version bounds of the ledger dependencies, the PVP (which we generally strive to follow) requires we also bump our major version if they added orphan instances eg. I haven't determined whether or not they did, so `Breaking` is conservation for that risk too.

I welcome any objections to using `Breaking` here.

Reply:
I don't see a problem with being conservative and using `Breaking` here :+1: